introspection-engine/connectors/introspection-connector/src/error.rs

```diff
@@ -19,13 +19,13 @@ impl ConnectorError {
         }
     }
 
-    pub fn url_parse_error(err: impl Display, url: &str) -> Self {
-        let details = user_facing_errors::quaint::invalid_url_description(url, &format!("{}", err));
+    pub fn url_parse_error(err: impl Display) -> Self {
+        let details = user_facing_errors::quaint::invalid_url_description(&err.to_string());
         let known = KnownError::new(InvalidDatabaseString { details });
 
         ConnectorError {
             user_facing_error: Some(known),
-            kind: ErrorKind::InvalidDatabaseUrl(format!("{} in `{}`)", err, url)),
+            kind: ErrorKind::InvalidDatabaseUrl(format!("{} in database URL", err)),
```

Comment:
Is `database URL` our preferred wording here?
Not connection string or connection URI?
Not datasource URL? (because `datasource.url` in schema)

Reply:
I don't have a definitive answer to that. I went with `URL` because of the `url = "..."` syntax in the datasource block.